registry.json

```diff
@@ -2430,5 +2430,20 @@
     "telegram": "devdertin",
     "operatorName": "Guillermo Céspedes",
     "github": "terostakepool"
+  },
+  {
+    "ticker": "ADATW",
+    "poolId": "7ce59da0bac7f64b40fcda60a7ee368d5ac8082d81f2bb75fef9e01e",
+    "social": {
+      "twitter": "",
+      "facebook": "",
+      "reddit": "",
+      "instagram": "",
+      "website": "",
+      "youtube": "https://www.youtube.com/channel/UCu--wp47a20YMJbrzJmtv2w"
+    },
+    "telegram": "Tw_JasonLai",
+    "operatorName": "linuxjason",
```

Comment:
We need a real name or leave it empty please.

Reply:
Please change to Jason thanks

Reply:
@linuxjason please can you change in the PR? Thanks